src/workerd/io/worker-entrypoint.c++

```diff
@@ -86,8 +86,9 @@ void WorkerEntrypoint::init(
   // IoContext, in which case we reuse it.
 
   auto newContext = [&]() {
-    auto actorRef = actor.map([](kj::Own<Worker::Actor>& ptr) -> Worker::Actor& {
-      return *ptr;
+    auto actorRef = actor.map([](Worker::Actor::LocalActorReference& ptr) -> Worker::Actor& {
+      return *KJ_REQUIRE_NONNULL(ptr.get());
+      // TODO(now): Might need to check this in more depth (could it be null??)
```

Comment:
This is a bit of a hack, but the PSW stuff in our internal repo required it IIRC. It's possible a bunch of tests are failing because of these `KJ_REQUIRE_NONNULL`s, I just haven't checked yet. Will take a closer look in the morning!